pkg/worker/handler/endpoint/ensure.go

```diff
@@ -12,8 +12,12 @@ func (h *Handler) Ensure() error {
 
 	for k, v := range mapping {
 		var url string
+		var exi bool
 		{
-			url = v[h.env.Environment]
+			url, exi = v[h.env.Environment]
+			if !exi {
+				continue
+			}
```

Comment:
I added the API case for our production CNAME, for which no other environment is configured. So this exception must be handled here.